resources/assets/lib/chat/input-box.tsx

```diff
@@ -73,11 +77,12 @@ export default class InputBox extends React.Component<Props> implements Dispatch
     this.focusInput();
   }
 
-  componentWillUnmount() {
-    dispatcher.unregister(this);
-  }
+  focusInput(always = true) {
+    const activeElement = document.activeElement;
+    if (!always && (activeElement instanceof HTMLInputElement || activeElement instanceof HTMLTextAreaElement)) {
```

Comment:
that's an attempt but there's nothing forcing focus on quick search input at the moment :eyes: 

(search thing -> open result in new tab with mouse (middle click etc) -> switch to the tab -> go back to chat tab)